packages/e2e_tests/cypress/e2e/step_definitions/Examination-timeTable/TC_14_Exam-table-footer-link.cy.js

```diff
@@ -0,0 +1,43 @@
+import { Then, And } from 'cypress-cucumber-preprocessor/steps';
+
+//Terms and Conditions page
+And('I click on Terms and conditions footer link', () => {
+	cy.clickFooterLink('Terms and conditions');
+});
+
+Then('I am on the Terms and conditions page', () => {
+	cy.pageVerification('Terms and conditions');
+	cy.go('back');
```

Comment:
Surly these back actions should not be part of the validation

Reply:
Back action is not part of the validation.